bindings/ocaml/lib/opendal.ml

```diff
@@ -18,5 +18,6 @@
 *)
 
 module Operator = Operator
+module Seek_from = Opendal_core.Seek_from
```

Comment:
Hi, does ocaml have native `Reader`, `IO` or `SeekFrom` concepts? How will ocaml's std lib handles File? On the other side, if a lib want to do read on a file, how will they call it?

Reply:
Yes, we can use `seek_command` from `Unix` package instead of `seek_from`

Reply:
Thanks a lot! I prefer to integrate with ocaml's native as close as possible.

Reply:
the OCaml seek API looks like `val lseek : file_descr -> (pos:int) -> seek_command -> int`, we can't use it directly because it is bound to fd. If we design the seek API to be similar to OCaml, it will be `operator -> int -> Unix.seek_command -> (int,string) result`. At the same time, we still need `seek_from `This middle layer handles the calls of rust and ocaml functions